src/ontoweaver/make_value.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import abc
+
+from . import errormanager
+from . import exceptions
+
+logger = logging.getLogger("ontoweaver")
+class ValueMaker(errormanager.ErrorManager, metaclass=abc.ABCMeta):
+    """
+    Interface for value makers that are used to define the extraction logic for each of the transformer classes.
+    """
+    def __init__(self, raise_errors: bool = True):
```

Comment:
Same rationale about `kwargs` than for `LabelMaker`: the constructor should take `kwargs`, not the call function, and there is no need for checking attributes.